extensions/amp-lightbox-gallery/0.1/amp-lightbox-gallery.js

```diff
@@ -433,10 +410,35 @@ export class AmpLightboxGallery extends AMP.BaseElement {
         toggle(dev().assertElement(this.descriptionBox_), true);
 
       }).then(() => {
-        this.vsync_.run({
-          measure: measureOverflowState,
-          mutate: mutateOverflowState,
-        }, {});
+        let descriptionOverflows, isInOverflowMode;
```

Comment:
Is the `state` argument not available using `measureMutateElement`? Or do you find it cleaner to use these closure variables? Just curious if part of the vsync API was missed in the `measureMutateElement` implementation.

Reply:
This is indeed no longer available in the new API. @jridgewell says there were too few users so he killed it.